tokens/connect/trezor.md

```diff
@@ -70,7 +70,7 @@ description: 通过本教程学习如何利用Moonbeam兼容以太坊的特性
 关于这5个以太坊账户，您可以跟随以下步骤进行操作：
 
  1. 选取您想要从Trezor设备导入的账户
- 2. 点击“解锁”
+ 2. 点击**解锁**
```

Comment:
**Unlock**

Reply:
please check the rest of the file for what should be changed but bold items shouldn't be translated